vllm/block.py

```diff
@@ -1,12 +1,36 @@
 """Token blocks."""
-from typing import List
+import weakref
+from collections import defaultdict
+from typing import Dict, List
 
 from vllm.utils import Device
 
 _BLANK_TOKEN_ID = -1
 
 DEFAULT_LAST_ACCESSED_TIME = -1
 
+TOKEN_BLOCKS = List[int]
+
+
+class BlockPool:
+    """A pool of physical blocks.
```

Comment:
Can you explain why we want to do this in the docstring?

Reply:
added in [99e7668](https://github.com/vllm-project/vllm/pull/5584/commits/99e766898a456ebd497fae5435045efa21b147f8)